src/style.css

```
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
```

```suggestion
  box-shadow: 0 2px 4px 0 #0000000d;
```
А что такое `#0000000d;`? в hex формате всего 6 символов

---

Написал в другом формате, взятом из Figma

---

Из макета в Figma